### ISSUE TITLE
NoSuchElementException: java.util.ArrayList$Itr.next

### ISSUE THREAD

**Issue by alschust** [tabulapdf/tabula#573]
Tabula API version: 1.1.0.16092117
Filename: WVU-48-new-tabula.pdf
Internal Server Error (500)

```
    Request Method:
    POST


    Request URL:
    http://127.0.0.1:8080/pdf/e4b9aac821e46cf68437cdde5baf1862ffdb26f5/data
```

[WVU-48-new-tabula.pdf](https://github.com/tabulapdf/tabula/files/551489/WVU-48-new-tabula.pdf)

**Comment by jeremybmerrill** [comment]
confirmed: 

````
Java::JavaUtil::NoSuchElementException: 
	java.util.ArrayList$Itr.next(java/util/ArrayList.java:854)
	java.util.Collections.min(java/util/Collections.java:635)
	technology.tabula.Page.getArea(technology/tabula/Page.java:67)
	technology.tabula.Page.getArea(technology/tabula/Page.java:107)
	java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:498)
````

**Comment by jeremybmerrill** [comment]
Should be fixed when the fix from #131 makes it into tabula-java and then into the next release of tabula. Closed for now.
